nilearn/decomposition/multi_pca.py

```diff
@@ -211,7 +230,7 @@ def fit(self, imgs, y=None, confounds=None):
             raise ValueError('Need one or more Niimg-like objects as input, '
                              'an empty list was given.')
         if confounds is None:
-            confounds = itertools.repeat(None, len(imgs))
+            confounds = [None] * len(imgs)  # itertools.repeat(None, len(imgs))
```

Comment:
I think that this is a minor difference, but I'd rather stay with the initial version if there's no reason to change it/